### PR TITLE
2047 merge unrelated to angular parts

### PR DESCRIPTION
**Pull request by RomanIovlev**

### CODE REVIEW

**Comment by mf-code** [comment]
[5481] - uuTCMmCO

**Comment by mf-code** [comment]
[5481] - ooqWzQPw

**Comment by mf-code** [comment]
[5481] - Allure report on Netlify: https://5ed3d2271c84485ec0327e21--musing-swartz-6fc060.netlify.app

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 49
- Added 6
           

Complexity increasing per file
==============================
- jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java  1
- jdi-bdd/src/main/java/com/epam/jdi/bdd/elements/CustomMenu.java  1
         

Complexity decreasing per file
==============================
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitDataListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitJListTests.java  -1
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitJListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitDataListTests.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753&bid=18389223)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+
+import static com.epam.jdi.light.elements.complex.table.Column.*;
+import static com.epam.jdi.light.elements.complex.table.TableMatcher.hasValue;
+import static com.epam.jdi.light.elements.complex.table.TableMatcher.*;
+import static com.epam.jdi.tools.StringUtils.*;
 import static io.github.com.StaticSite.*;
 import static io.github.com.pages.SimpleTablePage.*;
 import static io.github.epam.html.tests.site.steps.States.*;
 import static java.util.Arrays.*;
+import static org.hamcrest.Matchers.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:18]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package cucumberTests;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java [hunk 3]
```diff
@@ -1,16 +1,28 @@
 package io.github.epam.html.tests.elements.complex.table;
 
+import io.github.com.entities.Furniture;
 import io.github.epam.TestsInit;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import java.util.List;
+
+import static com.epam.jdi.light.elements.complex.table.Column.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:10]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.Column'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java [hunk 4]
```diff
@@ -1,16 +1,28 @@
...
+import static com.epam.jdi.light.elements.complex.table.TableMatcher.hasValue;
+import static com.epam.jdi.light.elements.complex.table.TableMatcher.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:12]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.TableMatcher'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java [hunk 5]
```diff
@@ -1,16 +1,28 @@
...
+import static com.epam.jdi.tools.StringUtils.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:13]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.StringUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

**Comment by mf-code** [comment]
[5486] - AHGYtTjP

**Comment by mf-code** [comment]
[5486] - Sw6hnLrc

**Comment by mf-code** [comment]
[5486] - Te7dvpvK

**Comment by mf-code** [comment]
[5486] - 7vMtNw1l

**Comment by mf-code** [comment]
[5486] - Allure report on Netlify: https://5ed50b8877d227a2f6379c55--musing-swartz-6fc060.netlify.app

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 49
- Added 6
           

Complexity increasing per file
==============================
- jdi-bdd/src/main/java/com/epam/jdi/bdd/elements/CustomMenu.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java  1
         

Complexity decreasing per file
==============================
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitDataListTests.java  -1
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitDataListTests.java  -1
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitJListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitJListTests.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753&bid=18389223)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:18]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 5 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:13]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.StringUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 3 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:10]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.Column'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 4 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:12]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.TableMatcher'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java [hunk 6]
```diff
@@ -1,16 +1,28 @@
...
 import static io.github.com.StaticSite.*;
 import static io.github.com.pages.SimpleTablePage.*;
 import static io.github.epam.html.tests.site.steps.States.*;
 import static java.util.Arrays.*;
+import static org.hamcrest.Matchers.*;
+import static org.jsoup.internal.StringUtil.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:19]: Codacy Issue found: [Avoid unused imports such as 'org.jsoup.internal.StringUtil'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

**Comment by mf-code** [comment]
[5487] - mz0cd85T

**Comment by mf-code** [comment]
[5487] - yvEkEZG2

**Comment by mf-code** [comment]
[5487] - KmELobKR

**Comment by mf-code** [comment]
[5487] - Up5QsJVR

**Comment by mf-code** [comment]
[5487] - FLuN4kEi

**Comment by mf-code** [comment]
[5487] - Allure report on Netlify: https://5ed5322387e8c73e103a8240--musing-swartz-6fc060.netlify.app

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 13
           

Complexity increasing per file
==============================
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java  1
- jdi-bdd/src/main/java/com/epam/jdi/bdd/elements/CustomMenu.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java  5
- jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java  1
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java  1
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java  1
         

Complexity decreasing per file
==============================
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitJListTests.java  -1
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitDataListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitJListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitDataListTests.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753&bid=18389223)

File: jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java [hunk 8]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests;
+package cucumberTests;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java [hunk 9]
```diff
@@ -1,7 +1,7 @@
-package cucmberTests;
+package cucumberTests;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:18]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java [hunk 10]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.steps;
+package cucumberTests.steps;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java [hunk 11]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.stepdefs;
+package cucumberTests.stepdefs;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java [hunk 12]
```diff
@@ -10,12 +10,12 @@
 
 import static com.epam.jdi.light.driver.WebDriverUtils.*;
 import static com.epam.jdi.light.elements.init.UIFactory.*;
-import static cucmberTests.test.data.TestData.*;
+import static cucumberTests.test.data.TestData.*;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java:13]: Codacy Issue found: [Avoid unused imports such as 'cucumberTests.test.data.TestData'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 6 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:19]: Codacy Issue found: [Avoid unused imports such as 'org.jsoup.internal.StringUtil'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 4 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:12]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.TableMatcher'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 5 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:13]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.StringUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java [hunk 13]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.test.data;
+package cucumberTests.test.data;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java (hunk 7 above)
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/User.java [hunk 14]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.test.data;
+package cucumberTests.test.data;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/User.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 3 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:10]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.Column'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

**Comment by mf-code** [comment]
[5491] - q40ZHulc

**Comment by mf-code** [comment]
[5491] - GL5s1CYD

**Comment by mf-code** [comment]
[5491] - I8eEJvff

**Comment by mf-code** [comment]
[5491] - 5KYDw4TV

**Comment by mf-code** [comment]
[5491] - 9HmrMOOg

**Comment by mf-code** [comment]
[5491] - Allure report on Netlify: https://5ed5ffc9079af3e79a13cb5a--musing-swartz-6fc060.netlify.app

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 13
           

Complexity increasing per file
==============================
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java  1
- jdi-bdd/src/main/java/com/epam/jdi/bdd/elements/CustomMenu.java  1
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java  5
- jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java  1
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java  1
         

Complexity decreasing per file
==============================
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitJListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitDataListTests.java  -1
+ jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitDataListTests.java  -1
+ test-examples/jdi-light-examples/src/test/java/io/github/epam/tests/google/WaitJListTests.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753&bid=18389223)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java (hunk 9 above)
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/User.java (hunk 14 above)
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/User.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java (hunk 11 above)
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java (hunk 7 above)
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 6 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:19]: Codacy Issue found: [Avoid unused imports such as 'org.jsoup.internal.StringUtil'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java (hunk 13 above)
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java (hunk 12 above)
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java:13]: Codacy Issue found: [Avoid unused imports such as 'cucumberTests.test.data.TestData'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 5 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:13]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.StringUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java (hunk 10 above)
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 3 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:10]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.Column'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:18]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java (hunk 4 above)
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/table/FurnitureTests.java:12]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.table.TableMatcher'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

File: jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java (hunk 8 above)
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5613753)

**Comment by mf-code** [comment]
[5493] - 3xZBfHAO

**Comment by mf-code** [comment]
[5493] - pYB2SUBE

**Comment by mf-code** [comment]
[5493] - lTbrywLb

**Comment by mf-code** [comment]
[5493] - Hrk1MkIU

**Comment by mf-code** [comment]
[5493] - qdIKA5VV

**Comment by mf-code** [comment]
[5493] - Allure report on Netlify: https://5ed6331ab8c25414a252bb1d--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5494] - DxCO5koq

**Comment by mf-code** [comment]
[5494] - swb0zSGK

**Comment by mf-code** [comment]
[5494] - zN6jAiBq

**Comment by mf-code** [comment]
[5494] - KYIC23cd

**Comment by mf-code** [comment]
[5494] - PKs0s1gR

**Comment by mf-code** [comment]
[5495] - IlrtuSZR

**Comment by mf-code** [comment]
[5495] - nNlCR5mW

**Comment by mf-code** [comment]
[5495] - ZQNkbPu4

**Comment by mf-code** [comment]
[5494] - Allure report on Netlify: https://5ed64de86afa3ca7929420d5--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5495] - HeVQsZ9q

**Comment by mf-code** [comment]
[5495] - YnVeJFTu

**Comment by mf-code** [comment]
[5495] - Allure report on Netlify: https://5ed6529aec7ef9ce0777e3f3--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5497] - fOXM7zU2

**Comment by mf-code** [comment]
[5497] - JctDfZ2m

**Comment by mf-code** [comment]
[5497] - 0lPRjgZc

**Comment by mf-code** [comment]
[5497] - xBhWp0cX

**Comment by mf-code** [comment]
[5497] - D6DXf827

**Comment by mf-code** [comment]
[5497] - Allure report on Netlify: https://5ed66395ec7ef9f28b77e41e--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5532] - Wj6eJnd1

**Comment by mf-code** [comment]
[5532] - cC4CrdhJ

**Comment by mf-code** [comment]
[5532] - 6yKLlwVV

**Comment by mf-code** [comment]
[5532] - 82ZW2sMl

**Comment by mf-code** [comment]
[5532] - c7wTTkx4

**Comment by mf-code** [comment]
[5532] - Allure report on Netlify: https://5ed8c1c668b3cf270d38b15e--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5551] - f2HbPaqN

**Comment by mf-code** [comment]
[5551] - 6MfQJLmA

**Comment by mf-code** [comment]
[5551] - Hg3uJ71f

**Comment by mf-code** [comment]
[5551] - dzvkRxdt

**Comment by mf-code** [comment]
[5577] - Ko3zFZ3y

**Comment by mf-code** [comment]
[5577] - CbDKCMKa

**Comment by mf-code** [comment]
[5577] - 32K3752n

**Comment by mf-code** [comment]
[5577] - MiGBzPxv

**Comment by mf-code** [comment]
[5578] - bbvzLP1L

**Comment by mf-code** [comment]
[5578] - j7w42Uzm

**Comment by mf-code** [comment]
[5578] - OTY0hfSy

**Comment by mf-code** [comment]
[5578] - yofTY0xU

**Comment by mf-code** [comment]
[5577] - Il6pF40p

**Comment by mf-code** [comment]
[5577] - Allure report on Netlify: https://5ede9a5f19fecce782cacd25--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5578] - jb8MxXuV

**Comment by mf-code** [comment]
[5578] - Allure report on Netlify: https://5ede9c1c2678e20246c41911--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5586] - tH0aCKMw

**Comment by mf-code** [comment]
[5586] - Ga7PeYjL

**Comment by mf-code** [comment]
[5586] - temMlpu1

**Comment by mf-code** [comment]
[5586] - 8YP9k03B

**Comment by mf-code** [comment]
[5586] - ZSHNaO7s

**Comment by mf-code** [comment]
[5586] - Allure report on Netlify: https://5edec90efef7101e2289f590--musing-swartz-6fc060.netlify.app

**Pull request merged**